demos/noise_suppression_demo/cpp/main.cpp

```diff
@@ -116,131 +117,152 @@ void read_wav(const std::string& file_name, std::vector<int16_t>& wave, RiffWave
         throw std::logic_error(error_msg + "for '" + file_name + "' file.");
     }
 
-    size_t wave_size = wave_header.data_length / 2;
+    size_t wave_size = wave_header.data_length / sizeof(int16_t);
     wave.resize(wave_size);
-    inp_wave.read((char*)&(wave.front()), wave_size*2);
+
+    inp_wave.read((char*)&(wave.front()), wave_size * sizeof(int16_t));
 }
 
-void write_wav(const std::string& file_name, const std::vector<int16_t>& wave, const RiffWaveHeader& wave_header) {
+void write_wav(const std::string& file_name, const RiffWaveHeader& wave_header, const std::vector<int16_t>& wave) {
     std::ofstream out_wave(file_name, std::ios::out|std::ios::binary);
     if(!out_wave.is_open())
         throw std::logic_error("fail to open " + file_name);
+
     out_wave.write((char*)&wave_header, sizeof(RiffWaveHeader));
-    out_wave.write((char*)&(wave.front()), wave.size()*2);
+    out_wave.write((char*)&(wave.front()), wave.size() * sizeof(int16_t));
 }
 
-int main(int argc, char *argv[]) {
+int main(int argc, char* argv[]) {
     try {
         // ------------------------------ Parsing and validating of input arguments --------------------------
         if (!ParseAndCheckCommandLine(argc, argv)) {
             return EXIT_FAILURE;
         }
 
+        slog::info << ov::get_openvino_version() << slog::endl;
+
         // Loading Inference Engine
-        InferenceEngine::Core ie;
+        ov::runtime::Core core;
 
-        InferenceEngine::CNNNetwork network = ie.ReadNetwork(FLAGS_m);
-        InferenceEngine::InputsDataMap inputs = network.getInputsInfo();
+        std::shared_ptr<ov::Model> model = core.read_model(FLAGS_m);
+        slog::info << "model file: " << FLAGS_m << slog::endl;
```

Comment:
```suggestion
        slog::info << "Reading model " << FLAGS_m << slog::endl;
```

Reply:
@anzhella-pankratova what you are trying to achieve with this change? The reading is continuous process, while we dump info for the model which already read (we have no physical capability to dump info during the reading is not it?).

Reply:
got it, changed